cli/client.go

```diff
@@ -943,6 +949,33 @@ func (c *viamClient) robotPart(orgStr, locStr, robotStr, partStr string) (*apppb
 	return nil, errors.Errorf("no machine part found for machine: %q part: %q", robotStr, partStr)
 }
 
+// getRobotPart wraps GetRobotPart API.
+// note: overlaps with viamClient.robotPart, which wraps GetRobotParts.
+// Use this variant if you don't know the robot ID.
+func (c *viamClient) getRobotPart(partID string) (*apppb.GetRobotPartResponse, error) {
```

Comment:
[nit] maybe call this `robotPartFromPartID` given your `note` above?

Reply:
this is wrapping the `GetRobotPart` api + the other one is wrapping the `GetRobotParts` api so there's already some built-in confusion here

afaict the semantics here are `GetThingSubthing` -> `viamClient.thingSubthing`, but in this case RobotParts is taking the RobotPart slot

no strong feeling about this + will change if you want

Reply:
Agree with Abe. I suspect that the other funcs should change to more closely match the API. Cli should be a simple wrapper over our api is possible